qiskit/qpy/binary_io/circuits.py

```diff
@@ -587,7 +626,7 @@ def _write_instruction(file_obj, instruction, custom_operations, index_map, use_
             not hasattr(library, gate_class_name)
             and not hasattr(circuit_mod, gate_class_name)
             and not hasattr(controlflow, gate_class_name)
-            and gate_class_name != "Clifford"
+            and gate_class_name not in ["Clifford", "AnnotatedOperation"]
```

Comment:
Do we not export `AnnotatedOperation` from `qiskit.circuit`? I feel like it should be there as it is a public interface.